src/status_im/data_store/realm/schemas/base/v2/bootnode.cljs

```diff
@@ -0,0 +1,8 @@
+(ns status-im.data-store.realm.schemas.base.v2.bootnode)
+
+(def schema {:name       :bootnode
+             :primaryKey :id
```

Comment:
Do we actually need synthetic `:id` as primary key, isn't for example `:address` totally sufficient as unique identifier ?

Reply:
not sure, I would feel safer to have a synthetic id, as we will allow to edit bootnodes addresses (in the next iteration), as it would make the update logic simpler, what do you reckon?

Reply:
I feel it's safer to actually enforce constraints/consistency at the db level, so for example when we feel that user should not be able to define two boot-nodes with different names but same addresses (or the combination of `name-address` should be unique), it should be expressed at the db level.
In the past I worked on code (user statuses for messages) where natural identifiers (`message-id/user-id`) where ignored for id and synthetic id was used instead and it was PITA to do migrations and reason about the code/consistency of data because you had to always count (in UI etc) with duplicates and so on.

Reply:
I feel this is slightly different scenario:

at the moment there's no problem having 2 bootnodes with the same address (no consequence to it), and we are not going to enforce uniqueness on the address nor name (advanced settings).

Updating with a synthetic id is trivial, updating the address being an *editable* unique identifier means that it requires a delete/insert everytime, rather then a replace, also would implicitly makes the field unique, which at the moment we are not going to handle via UX.

I agree that if it needs to be unique and non-editable (seems like the case with `user-statuses` in your example) using a field makes things simpler, but here it would be an *editable* unique id.

I am happy to backoff if you feel strongly about it, but to me seems like it would not provide benefits.

Reply:
No very strong feelings about it and your arguments are actually convincing, this is little different thing, approving.